src/typography/print/index.md

```diff
@@ -8,5 +8,5 @@ title: Print
 The recommended sizes for common document formats are illustrated here.
 {% sectionHighlight { classes: "dark-magenta" } %}
 
-![TODO](./print-guides.svg)
+![Diagram of A4-sized letterhead and letter. Labels show the minimum sizes for the logo (10mm height), header text (16pt) and body text (12pt).](./print-guides.svg)
```

Comment:
The information about the specific sizes still needs to be in the body copy. Or would you disagree with that?

Reply:
If/when you move that into the body copy, the alt text wouldn't need to be that descriptive either and you could cut it a bit.